### ISSUE TITLE
add dpi info to dimensions.json

### ISSUE THREAD

**Issue by xristy** [buda-base/volume-manifest-builder#33]
It would be useful to add dpi per image to dimensions.json to help with handling of [mixed resolution images](https://github.com/buda-base/library-issues/issues/280).

**Comment by jimk-bdrc** [comment]
@eroux @xristy Required timeframe and urgency?

**Comment by jimk-bdrc** [comment]
Backlog vs new works. Usefulness?

**Comment by jimk-bdrc** [comment]
How about this?
```json
{
    "filename": "I8LS682310224.jpg",
    "width": 1400,
    "height": 2128,
    "dpi": [
      400,
      400
    ]
  }
```

**Comment by jimk-bdrc** [comment]
Fixed in #36
